package.json

```diff
@@ -33,5 +33,8 @@
     "ts-node": {
       "typescript": "$typescript"
     }
+  },
+  "dependencies": {
+    "@mediapipe/tasks-vision": "^0.1.0-alpha-12"
```

Comment:
Hey @mayarajan3 -- one note, when you install packages for an extension, you should make sure to run `pnpm add` at that extension's location, so each extension can effectively have it's own version of that package. Leads to some duplication in bundle size for sure, but makes our lives of supporting extensions a lot easier (that each extension has it's own, independent dependencies)

Reply:
I will change this in an upcoming commit (as I try to diagnose some of the build errors we're having)